src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -17,16 +17,16 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
-  <enum name="Privilege" type="ENUM8">
+  <enum name="AccessControlEntryPrivilegeEnum" type="ENUM8">
     <cluster code="0x001F"/>
     <item name="View" value="0x01"/>
-    <item name="ProxyView" value="0x02"/>
+    <item name="Proxy View" value="0x02"/>
```

Comment:
It's a bit pity the spec isn't consistent in naming types. Some enums use CamelCase for item names, some add a space between the words...

Reply:
Yes, the fact that the naming in the spec is pretty inconsistent is not great.  It's even worse for "URL" vs "Url".... :(